test/Microsoft.AspNet.WebHooks.Custom.SqlStorage.Test/Extensions/HttpConfigurationExtensionsTests.cs

```diff
@@ -22,5 +22,40 @@ public void Initialize_SetsStore()
             // Assert
             Assert.IsType<SqlWebHookStore>(actual);
         }
+
+        [Theory]
+        [InlineData("Default")]
+        [InlineData("")]
+        [InlineData(null)]
+        public void Initialize_SetsStore_WithCustomConnectionString(string nameOrConnectionString)
+        {
+            // Arrange
+            HttpConfiguration config = new HttpConfiguration();
+
+            // Act
+            config.InitializeCustomWebHooksSqlStorage(true, nameOrConnectionString);
+            IWebHookStore actual = CustomServices.GetStore();
+
+            // Assert
+            Assert.IsType<SqlWebHookStore>(actual);
```

Comment:
Add a `string` parameter for the `expectedNameOrConnectionString` and `Assert.Equal(expectedNameOrConnectionString, actual.NameOrConnectionString)`. You may need to use the `IsType<T>(...)` return value e.g.
``` c#
var actualStore = Assert.IsType<SqlWebHookStore>(actual);
Assert.Equal(expectedNameOrConnectionString, actualStore.NameOrConnectionString);
```

Reply:
But `SqlWebHookStore` doesn't have public property `NameOrConnectionString`, there is:
```C#
private readonly string _nameOrConnectionString;
```